web/src/components/tasks/business-formation/contacts/ContactsStep.test.tsx

```diff
@@ -209,6 +210,108 @@ describe("Formation - ContactsStep", () => {
     });
   });
 
+  describe("when nonprofit", () => {
+    const legalStructureId = "nonprofit";
+
+    const attemptApiSubmission = async (page: FormationPageHelpers): Promise<void> => {
+      await page.stepperClickToReviewStep();
+      await page.clickSubmit();
+      await page.stepperClickToContactsStep();
+    };
+
+    it("auto-fills fields from userData if it exists", async () => {
+      const members: FormationMember[] = [
+        {
+          name: "Joe Biden",
+          addressCity: "Washington",
+          addressLine1: "1600 Pennsylvania Ave NW",
+          addressLine2: "Office of the President",
+          addressState: { name: "District of Columbia", shortCode: "DC" },
+          addressCountry: "US",
+          businessLocationType: "US",
+          addressZipCode: "20500",
+        },
+      ];
+      const incorporators: FormationIncorporator[] = [
+        {
+          name: "Donald Whatever",
+          addressCity: "Miami",
+          addressLine1: "160 Something Ave NW",
+          addressLine2: "Office of Whatever",
+          addressState: { name: "Florida", shortCode: "FL" },
+          addressCountry: "US",
+          addressZipCode: "20501",
+          businessLocationType: "US",
+          title: "Incorporator",
+          signature: true,
```

Comment:
I don't think this is a huge deal, but given this is an open source project that others will likely see and could fork we should be careful about making political remarks.

Reply:
Lol, a good point but should be a follow up. This is already in the codebase, it's not a new addition from this PR

Reply:
Follow up out of this PR